packages/@react-spectrum/tooltip/test/TooltipTrigger.test.js

```diff
@@ -65,22 +61,17 @@ describe('TooltipTrigger', function () {
         </Provider>
       );
 
-      expect(() => {
-        getByRole('tooltip');
-      }).toThrow();
-
       let button = getByRole('button');
       triggerPress(button);
 
       let tooltip = getByRole('tooltip');
 
-      // wait for appearance
       await wait(() => {
         expect(tooltip).toBeVisible();
       });
     });
 
-    it('pressing esc should close the tooltip after a click event', async function () {
+    it('pressing escape should close the tooltip after a click event', async function () {
```

Comment:
why does this one check toBeInTheDocument instead of toBeVisible?

Reply:
why does this one use await tooltip toBeInTheDocument && await waitForDomChange? why don't any of the others use waitForDomChange or why isn't waitForDomChange replaced with await tooltip not toBeInTheDocument?

Reply:
I wasn't sure which was the best to use. 
I pushed a change to use inDocument / notInDocument instead any visibility checks.
Do you have a preference for the waitForDomChange, etc ?

Reply:
I'm resolving this comment for now given the work we did yesterday to try and switch to fake timers. Once we have that in place, we should be able to sort this. But that'll be a later PR